web/styles/input_pill.css

```diff
@@ -143,6 +143,7 @@
 
 #invitee_emails_container .pill-container {
     width: 100%;
+    background-color: hsl(0deg 0% 100%);
```

Comment:
This is probably much better done with a (usually existing) color variable; in general we're trying to move away from that model, and in particular towards not needing `dark_theme.css` to do anything other than control color variables.

Reply:
There is one variable that matches the requirement but seems unusable (`--color-compose-collapsed-reply-button-area-background`) so added a custom one.